pkgs/tools/misc/rauc/default.nix

```
@@ -1,14 +1,19 @@
-{ autoreconfHook
-, curl
+{ curl
 , dbus
 , fetchFromGitHub
+, fetchpatch
 , glib
 , json-glib
 , lib
 , nix-update-script
 , openssl
 , pkg-config
 , stdenv
+, meson
+, ninja
+, utillinux
```

```suggestion
, util-linux
```
this is an alias and should not be used in nixpkgs

---

I am not sure why this is not triggering an error for ofborg

---

Fixed.